velox/functions/prestosql/benchmarks/IntArrayProxyBenchmark.cpp

```diff
@@ -206,6 +206,25 @@ class ArrayProxyBenchmark : public functions::test::FunctionBenchmarkBase {
     folly::doNotOptimizeAway(cnt);
   }
 
+  size_t runStdRef() {
+    for (auto k = 0; k < 100; k++) {
+      std::vector<std::vector<std::optional<int>>> arrayVector;
```

Comment:
should this be `std::vector<std::optional<std::vector<std:::optional<int>>>>`, since the arrays can also be null?